mermaid.Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Build sysl-catalog binary
+FROM golang:alpine AS builder
+ADD . /src
+RUN cd /src && CGO_ENABLED=0 go build -o sysl-catalog
+
+FROM chromedp/headless-shell:latest
+WORKDIR /usr
+COPY --from=builder /src/sysl-catalog /bin/
+
+ENTRYPOINT [ "sysl-catalog" ]
+
```

Comment:
Is there a reason why we need another dockerfile?

Reply:
I didn't want to break the existing dockerfile, in case it's still needed for current sysl-catalog users. I'll be removing the existing file once we've migrated any consumers off the old docker image

Reply:
https://docs.docker.com/engine/reference/builder/#arg

Reply:
Awesome, didn't know that was a thing! I'll keep that in mind, although it seems simpler to me to keep them as separate files as we can just delete the old Dockerfile when we deprecate PlantUML